### PR TITLE
Small fixes and improvements in models.py

### PR DESCRIPTION
**Pull request by lbarcziova**
I was testing my migration script and found some bugs in representations (calling non-existing attributes)

### CODE REVIEW

File: packit_service/models.py [hunk 1]
```diff
@@ -840,3 +836,9 @@ def create(cls, event):
                 ]
                 session.add(installation)
             return installation
+
+    def __repr__(self):
+        return f"InstallationModel(id={self.id}, account={self.account_login})"
+
+    def __str__(self):
+        return self.__repr__()
```
Comment (jpopelka) [inline packit_service/models.py:844]: `__str__` is I think not needed if it's the same `__repr__`.
From [docs](https://docs.python.org/3/reference/datamodel.html#object.__repr__):
"If a class defines `__repr__()` but not `__str__()`, then `__repr__()` is also used when an “informal” string representation of instances of that class is required."

File: packit_service/models.py [hunk 2]
```diff
@@ -245,7 +245,9 @@ def get_or_create(
             return git_branch
 
     def __repr__(self):
-        return f"GitBranchModel(id={self.pr_id}, name={self.name},  project={self.project})"
+        return (
+            f"GitBranchModel(id={self.id}, name={self.name},  project={self.project})"
```
Comment (lachmanfrantisek) [inline packit_service/models.py:249]: We probably don't need the private keys in that.

**Comment by lbarcziova** [comment]
@lachmanfrantisek I had not noticed that, sorry

File: packit_service/models.py (hunk 1 above)
Comment (lbarcziova) [reply packit_service/models.py:844]: It is used for every class in models, so can I remove it everywhere?

**Comment by lachmanfrantisek** [comment]
> @lachmanfrantisek I had not noticed that, sorry

No need to sorry, I discovered that during the work on some fix and haven't created an issue. The simplification of `create*` methods should be still valid. I hope it will be merged soon so you can rebase.

File: packit_service/models.py (hunk 1 above)
Comment (TomasTomecek) [reply packit_service/models.py:846]: @jpopelka you're idd correct - `__repr__` is better and we don't need `__str__`:
```python
class C:
    a = 1
    b = "2"

    def __str__(self):
        return f"C(a={self.a}, b={self.b})"


class D:
    a = 1
    b = "2"

    def __repr__(self):
        return f"D(a={self.a}, b={self.b})"


print(f"str(C()): {str(C())}")
print(f"str(C()): {C()}")
print(f"str(D()): {str(D())}")
print(f"str(D()): {D()}")
print(f"repr(C()): {repr(C())}")
print(f"repr(D()): {repr(D())}")
```

```
str(C()): C(a=1, b=2)
str(C()): C(a=1, b=2)
str(D()): D(a=1, b=2)
str(D()): D(a=1, b=2)
repr(C()): <__main__.C object at 0x7fae2cce9fd0>
repr(D()): D(a=1, b=2)
```

**Pull request merged**